node/src/BaseClient.ts

```diff
@@ -506,6 +506,67 @@ export type ReadFrom =
 
 /**
  * Configuration settings for creating a client. Shared settings for standalone and cluster clients.
+ *
+ * @remarks
```

Comment:
I think we shuold have this for all clients.

Reply:
can you elaborate? @Yury-Fridlyand

Reply:
We need such description for docs for client config for all our clients